.travis.yml

```diff
@@ -10,54 +10,64 @@ stages:
 jobs:
   include:
     - stage: test
+      name: osx
       os: osx
       before_install:
         - SKIP=`.travis/check-mergify-merge` && if [[ "$SKIP" = "skip" ]]; then exit 0; fi
         - SKIP=`.travis/check-change '^(docs|spec)/'` && if [[ "$SKIP" = "skip" ]]; then exit 0; fi
-      install:
         - SKIP=`.travis/check-change '^test/'` && if [[ "$SKIP" != "skip" ]]; then RUN_UNIT_TEST=1; fi
```

Comment:
Could you move it to `before_install`? and `"$SKIP" = "noskip"` is more natural.